src/components/interfaces/MOBILE_API.xml

```diff
@@ -5363,6 +5376,39 @@
      </param>
   </function>
 
+  <function name="SetAudioStreamingIndicator" functionID="SetAudioStreamingIndicatorID" messagetype="request">

```

Comment:
@dtrunov we have no firmating convention abut xml, but please remove redundant space there or add new one to https://github.com/smartdevicelink/sdl_core/pull/1233/files#diff-cfd2a12866985b9fdcfdbc6ee134baf1R5388 line